.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseTimeExtractor.cs

```diff
@@ -90,6 +90,13 @@ private List<Token> BasicRegexMatch(string text)
 
                 foreach (Match match in matches)
                 {
+                    // Check that the extracted time is not part of a decimal number (e.g. 123.24)
```

Comment:
Shouldn't this have been removed after the YAML changes?

Reply:
yes, removed now.